packages/core/src/server/options.ts

```diff
@@ -7,7 +7,11 @@ export const apiOptions: APIOptions = {
   account: storeConfig.api.storeId,
   environment: storeConfig.api.environment as APIOptions['environment'],
   subDomainPrefix: storeConfig.api.subDomainPrefix ?? ['www'],
-  hideUnavailableItems: storeConfig.api.hideUnavailableItems,
+  searchOptions: {
+    hideUnavailableItems: storeConfig.api.hideUnavailableItems,
+    simulationBehavior: (storeConfig.api as Record<string, any>)
```

Comment:
I think we can create a type of `faststore.config` to avoid mistakes. It can be another task.